.github/workflows/test_published_images.yml

```diff
@@ -37,6 +37,8 @@ jobs:
       - name: Parse input into matrix
         id: set-matrix
         run: |
+          . .github/scripts/workflow-commands.functions.sh
```

Comment:
does this need `# shellcheck....`

Reply:
Shellcheck doesn’t (natively) work on non-shell scripts (ie those embedded in other stuff). If you use something like action lint it would, but I don’t think anyone is so not sure of the value of adding here.

Reply:
Understood. But...

Here we have the spellcheck comment
**https://github.com/hazelcast/hazelcast-docker/blob/87cab87ce9ef71c7aff0047a6abf63c305de9cbb/.github/workflows/test_published_images.yml#L131**

And not here in the same file:
https://github.com/hazelcast/hazelcast-docker/blob/87cab87ce9ef71c7aff0047a6abf63c305de9cbb/.github/workflows/test_published_images.yml#L40

Both are non-shell

Reply:
Good spot. Personally I don’t _really_ mind which way it is, but I guess if I’m changing it it should be consistent. Any opinions either way?

Reply:
I think lets add even if it is benign.

Reply:
https://github.com/hazelcast/hazelcast-docker/pull/842/commits/d435d7f22a4c0972bf3f6dd6db3f4afdbdab5432